pages/imprint.vue

```diff
@@ -58,44 +60,50 @@
       </p>
       <p>
         Quelle:
-        <a
-          href="https://www.ortner-rechtsanwalt.at/strafrecht-festnahme-verhaftung/"
+        <NuxtLink
+          to="https://www.ortner-rechtsanwalt.at/strafrecht-festnahme-verhaftung/"
+          target="_blank"
           class="underline decoration-solid underline-offset-4"
-          >Rechtsanwalt Wien Festnahme</a
+          >Rechtsanwalt Wien Festnahme</NuxtLink
         >
       </p>
       <p>
         Schriftarten von
-        <a
-          href="https://fonts.google.com/"
+        <NuxtLink
+          to="https://fonts.google.com/"
+          target="_blank"
           class="underline decoration-solid underline-offset-4"
-          >Google Fonts</a
+          >Google Fonts</NuxtLink
         >.<br />
         Icons von
-        <a
-          href="https://iconify.design/"
+        <NuxtLink
+          to="https://iconify.design/"
+          target="_blank"
           class="underline decoration-solid underline-offset-4"
-          >iconify</a
+          >iconify</NuxtLink
         >.<br />
         Bilder von
-        <a
-          href="https://unsplash.com/"
+        <NuxtLink
+          to="https://unsplash.com/"
+          target="_blank"
           class="underline decoration-solid underline-offset-4"
-          >unsplash</a
+          >unsplash</NuxtLink
         >,
-        <a
-          href="https://pixabay.com/"
+        <NuxtLink
+          to="https://pixabay.com/"
+          target="_blank"
           class="underline decoration-solid underline-offset-4"
-          >pixabay</a
+          >pixabay</NuxtLink
         >.
       </p>
     </div>
-    <a
-      href="/#about"
+    <NuxtLink
+      to="/#about"
+      :external="true"
```

Comment:
warum ist das external?
siehe https://nuxt.com/docs/api/components/nuxt-link

Reply:
dein link hat mir leider nichts gebracht, da ich auf einen hash auf eine andere page verweise (imprint.vue -> index.vue#about) und das nicht funktioniert hat bei mir wies in der documentation steht
deshalb ein bisschen zusammengeflickt mit der discussion hier: https://github.com/nuxt/nuxt/discussions/15995
also so wies jetzt ist funktioniert es, vlt gibts was schöneres dann gerne anmerken

Reply:
Ich denk es ist aber besser wir scrollen einfach nicht dorthin als wir lassen die seite komplett neu laden

Reply:
ich weiß ned, finds komisch wenn man vom impressum dann wieder ganz rauf kommt, deswegen wollte ich das eigentlich so lösen aber könnens weggeben